bambi/backend/pymc.py

```diff
@@ -258,15 +267,24 @@ def _run_mcmc(
             bx_sampler = operator.attrgetter(sampler_backend)(
                 bx_model.mcmc  # pylint: disable=no-member
             )
-            idata = bx_sampler(seed=jax_seed, **kwargs)
+            idata = bx_sampler(
```

Comment:
looks fine to me -- `bayeux` will silently ignore unused kwargs (feel free to open an issue if you hate this -- i have no strong feelings!).

note that a design quirk i feel a little strongly about is that `bayeux` uses the argument names from underlying libraries, so it is often `num_chains` for example. 

not sure what the solution is! bayeux will tell you what keyword arguments each method accepts...

Reply:
> looks fine to me -- bayeux will silently ignore unused kwargs (feel free to open an issue if you hate this -- i have no strong feelings!).

Aaaaaah ok, now I understand why no error was thrown

> note that a design quirk i feel a little strongly about is that bayeux uses the argument names from underlying libraries, so it is often num_chains for example.

Ok so that means this PR doesn't solve the issue for libraries using `num_chains` instead of `chains` instead?

Reply:
right -- you could special case things (the only samplers are from `blackjax`, `numpyro`, `nutpie`, `tfp`, `flowMC`), which is a little brittle, but would probably cover most cases. 

here's a snippet: 
```python
for method in model.mcmc.methods:
  sampler = getattr(model.mcmc, method)
  print(method)
  for v in ('chains', 'num_chains', 'num_draws', 'num_samples', 'num_results'):
    if any(k == v for step in sampler.get_kwargs().values() for k in step):
      print('\t', v)
```

it looks like the colab i'm running this in doesn't have flowMC or nutpie, but the output is 
```
tfp_hmc
	 num_chains
	 num_draws
tfp_nuts
	 num_chains
	 num_draws
tfp_snaper_hmc
	 num_chains
	 num_results
blackjax_hmc
	 num_chains
	 num_draws
blackjax_chees_hmc
	 num_chains
	 num_draws
blackjax_meads_hmc
	 num_chains
	 num_draws
blackjax_nuts
	 num_chains
	 num_draws
blackjax_hmc_pathfinder
	 num_chains
	 num_draws
blackjax_nuts_pathfinder
	 num_chains
	 num_draws
numpyro_hmc
	 num_chains
	 num_samples
numpyro_nuts
	 num_chains
	 num_samples
```

Reply:
Thanks @ColCarroll ! That will indeed do it, but my concern is: how often do you add / remove samplers on Bayeux? Because this solution would force Bambi to update and release each time Bayeux adds / removes samplers